### PR TITLE
fix: create LICENSE

### PR DESCRIPTION
**Pull request by jonaslagoni**
Adds license file

### CODE REVIEW

**Comment by jonaslagoni** [comment]
@derberg @fmvilas 

Should I add this directly to the repository?

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (fmvilas) [inline LICENSE:189]:
```suggestion
   Copyright 2021 AsyncAPI Initiative
```

**Review by fmvilas — COMMENTED** [review]
Not sure you should be adding it in this repo or it's ok to have it on the `.github` one only. @derberg may be able to answer this question better.

**Comment by jonaslagoni** [comment]
Alright, it is just GitHub complaining the file is missing 🤷‍♂️

**Comment by derberg** [comment]
License has to be in every repo and must be a part of the published artifact. Big companies cannot use software otherwise. I had this at SAP, we could not use library if license was not in repo, and then we had also to make sure the license is also in the published package

**Comment by jonaslagoni** [comment]
Alright, let's add it then 😄

**Review by derberg — APPROVED** [review]

**Comment by derberg** [comment]
I recommend changing to `fix` to release, as starting from the next release anybody can use it without any issues

**Pull request merged**